build_openroad.sh

```diff
@@ -358,19 +359,29 @@ __common_setup()
         fi
 }
 
+__logging()
+{
+        mkdir -p "${buildDir}"
+        exec > >(tee -i ${buildDir}/build-$(date +%s).log)
```

Comment:
Hmmm, I was only thinking about saving the most recent build log. It seems like most users won't need to save multiple logs. On the other hand, saving logs doesn't hurt much. @vvbandeira can decide.